Contest_API.md

```diff
@@ -1337,13 +1440,20 @@ zip archive. These must be stored directly from the root of the zip
 file, i.e. there must not be extra directories (or files) added unless
 these are explicitly part of the submission content.
 
-#### POST and PUT submissions
+#### Submission capabilities 
+
+To add a submission clients can use the `POST` method on the submissions endpoint or the
+`PUT` method directly on an object url. One of the following [capabilities](#capabilities)
+is required to add submissions:
 
-To add a submission one can use the `POST` method on the submissions endpoint or the
-`PUT` method directly on an object url. `POST` is typically used by teams submitting
-to the contest and `PUT` is used by admin users or tools.
-Both must include a valid JSON object with the same properties the submission
-endpoint returns with a `GET` request with the following exceptions:
+| Name              | Description
+| :---------------- | :----------
+| team_submit       | POST a submissision as a team
+| proxy_submit      | POST a submission as a proxy (able to submit on behalf of a team)
+| admin_submit      | POST or PUT a submission as an admin
```

Comment:
maybe add something along "This allows overriding submission id, team and time among other things"? Or refer to the explanation below on how this can be used?

Reply:
Added "descriptions below" above both tables.